Update new-shield.md

chadbailey59 opened the pull request:
I got tripped up by misreading some docs, so I figured I'd try to clarify things a bit... and while I'm at it, shorten up the example keyboard name to avoid potential bluetooth name length problems. :)

caksoylar commented on docs/docs/development/new-shield.md:
```diff
@@ -48,16 +48,16 @@ shield to get it picked up for ZMK, `Kconfig.shield` and `Kconfig.defconfig`.
+This will make sure that a new configuration value named `SHIELD_MY_BOARD` is set to true whenever `my_board` is added as a shield in your `build.yaml` file. This configuration value will be used in Kconfig.defconfig to set other properties about your shield, so make sure that you use the same name for the configuration value in both places. 
 
-**For split boards**, you will need to add configurations for the left and right sides.
+**For split boards**, you will need to add configurations for the left and right sides, and check for the sibling you're building. For example, if your board's `id` is `my_board` and your `siblings` are named `my_board_left` and `my_board_right`, it would look like this:
```
```suggestion
**For split boards**, you will need to add configurations for the left and right sides. For example, if your split halves are named `my_board_left` and `my_board_right`, it would look like this:
```

I had trouble understanding "and check for the sibling you're building" so I suggest removing it, let me know if I am missing something.

Again I suggest removing the reference to metadata fields because they haven't been mentioned yet. I'd appreciate any feedback on this from other reviewers though. The distinction between board `id` and the names of the `sibling`s (which are the actual shield names used when building) aren't very clear at this point which is not ideal.

caksoylar reviewed (changes requested):
Thank you for opening this PR! I have a few improvement suggestions that I think makes things better in the context of the page.

caksoylar commented:
One more thing, we would want to change the keyboard name config to the shorter version for the split example as well on line 101.

chadbailey59 commented on docs/docs/development/new-shield.md:
```diff
@@ -48,16 +48,16 @@ shield to get it picked up for ZMK, `Kconfig.shield` and `Kconfig.defconfig`.
 ### Kconfig.shield
 
-The `Kconfig.shield` file defines any additional Kconfig settings that may be relevant when using this keyboard. For most keyboards, there is just one additional configuration value for the shield itself, e.g.:
+The `Kconfig.shield` file defines any additional Kconfig settings that may be relevant when using this keyboard. For most keyboards, there is just one additional configuration value for the shield itself. If your YAML file sets your `id` to a value of `my_board`, it would look like this:
```
👍

chadbailey59 commented:
Thanks for taking a look! I accepted your proposed changes, removed the line you suggested, and updated the keyboard name in the split configuration as well.

dxmh reviewed (approved):
Thanks @chadbailey59 and @caksoylar! 👍
